cmd/tailscale/cli/status.go

```diff
@@ -127,6 +127,11 @@ func runStatus(ctx context.Context, args []string) error {
 		return err
 	}
 
+	if st.BackendState == ipn.Stopped.String() {
+		fmt.Println("tailscaled has been stopped, use 'tailscale up' to start")
```

Comment:
It's not called "tailscaled" on Mac or Windows, and "tailscale up" isn't supported except on Linux. (But the CLI exists on all platforms)

So let's just say:

    "Tailscale is stopped."

(I'm avoiding saying "is not running", because the process is running, which could confuse people... hence saying "is stopped")